worker/pkg/workflows/datasync/workflow/workflow.go

```diff
@@ -623,3 +623,11 @@ func getAccountStatusTimerDuration() time.Duration {
 	}
 	return time.Duration(envtime) * time.Second
 }
+
+func getTableSyncMaxConcurrency() int {
+	maxConcurrency := viper.GetInt("TABLESYNC_MAX_CONCURRENCY")
+	if maxConcurrency == 0 {
```

Comment:
Best to change this to `<=0` to prevent people from setting it to something like `-1`